schemas/context/namespace.description.md

```diff
@@ -0,0 +1,15 @@
+The namespace ensures first, second, and third party user and device identities can be used in customer engagement platforms.
```

Comment:
Is this the only use case for a namespace?

Reply:
Yes